h2o-py/h2o/backend/__init__.py

```diff
@@ -48,10 +48,13 @@
 from distutils.version import StrictVersion
 import sys
 
-if StrictVersion(colorama.__version__) < StrictVersion("0.3.8"):
+if not hasattr(colorama, '__version__'):
+    print("[WARNING] H2O requires colorama module of version 0.3.8 or newer. You have older version .\n"
```

Comment:
Should we fail directly?

btw: in theory, setup.py for H2O python package should require version >= 0.3.8.

Reply:
It is already there